src/commands/compile_raw.rs

```diff
@@ -36,10 +36,23 @@ pub fn compile(sierra_program: Value) -> Result<Value> {
     )?;
     let assembled_cairo_program = cairo_program.assemble();
 
+    // taken from cairo compiler.
+    let builtins = vec![
+            "pedersen_builtin",
+            "range_check_builtin",
+            "bitwise_builtin",
+            "ec_op_builtin",
+            "poseidon_builtin",
+        ];
+    
+    let entry_point = calculate_entry_point(&sierra_program, "::main")?;
```

Comment:
I don't think a tool that does sierra -> casm compilation should return this information. It should be read from sierra directly by your tool. Alternatively when new sierras come up that have a different format a separate tool for extracting entry point statement idx should be created

Reply:
The motivation was for the VM to be agnostic of sierra if possible in case sierra is not required by the VM aside from this computation.

Reply:
I agree with @piotmag769 - this should not be passed here if it's available already in the sierra file. It's basically duplicating the information here

Reply:
>The motivation was for the VM to be agnostic of sierra if possible in case sierra is not required by the VM aside from this computation.

I agree that the VM should be sierra agnostic, but USC is not the kind of tool you would use for extracting entrypoint info from sierra since it has nothing to do with the compilation.

Reply:
A separate tool, sth like `universal-sierra-entrypoint-fetcher` (didn't think much about the name) would be good for that